src/main/java/frc/robot/config/FluidConstant.java

```diff
@@ -76,14 +76,6 @@ public T getValue() {
         return this.value;
     }
     
-    /**
-     * Get method for implementing the Supplier interface
-     * @return The current value of this constant
-     */
-    public T get() {
-        return this.value;

```

Comment:
This issue is fixed in master. Can you revert the changes to this file so we don't break it again?

Reply:
Rebasing should simply remove this problem, no need to explicitly revert anything.